frontend/src/components/core/Sidebar/Sidebar.test.tsx

```diff
@@ -104,7 +104,7 @@ describe("Sidebar Component", () => {
 
     expect(wrapper.find("StyledSidebarContent")).toHaveStyleRule(
       "overflow",
-      "overlay"
+      "auto"
```

Comment:
This can be reset to `"overlay"` given the `overflow` property change and should still pass the tests 👍🏼

Reply:
SGTM!